editor/components/CrashMapCard.tsx

```diff
@@ -42,11 +49,17 @@ export default function CrashMapCard({
     mapRef.current?.resize();
   });
 
-  const [isEditingCoordinates, setIsEditingCoordinates] = useState(false);
-  const [editCoordinates, setEditCoordinates] = useState<LatLon>({
-    latitude: 0,
-    longitude: 0,
+  const [isEditing, setIsEditing] = useState(false);
+  const [mapLatLon, setMapLatLon] = useState<LatLon>({
+    latitude: DEFAULT_MAP_PAN_ZOOM.latitude,
+    longitude: DEFAULT_MAP_PAN_ZOOM.longitude,
   });
+  const [formLatLon, setFormLatLon] = useState<LatLonString>({
+    latitude: String(DEFAULT_MAP_PAN_ZOOM.latitude),
+    longitude: String(DEFAULT_MAP_PAN_ZOOM.longitude),
+  });
```

Comment:
now we have a pair of state variables that manage the lat/lon coordinates. a numeric interface for the map, and a string-based on for the form inputs